project-keeper/src/main/java/com/exasol/projectkeeper/validators/dependencies/renderer/DependencyPageRenderer.java

```diff
@@ -91,7 +91,11 @@ private String renderDependencyName(final ProjectDependency dependency,
     }
 
     private String renderLicense(final License license, final MarkdownReferenceBuilder markdownReferenceBuilder) {
-        return renderLink(license.getName(), license.getUrl(), markdownReferenceBuilder);
+        String name = license.getName();
+        if (Workarounds.ALTERNATING_DEPENDENCIES.isActive()) {
+            name = name.replace("The Apache Software License", "Apache License");
+        }
```

Comment:
Optional: You could move this to `Workarounds.ALTERNATING_DEPENDENCIES.getLicenseName(license)`.
Then the logic is centralized in a single place.

Reply:
👍